src/debugger/nodeDebugWrapper.ts

```diff
@@ -250,4 +252,21 @@ export class NodeDebugWrapper {
     private isNullOrUndefined(value: any): boolean {
         return typeof value === "undefined" || value === null;
     }
+
+    /**
+     * Parses settings.json file for workspace root property
+     */
+    private getProjectRoot(args: any): string {
+        try {
+            let vsCodeRoot = path.resolve(args.program, "../..");
+            let settingsPath = path.resolve(vsCodeRoot, ".vscode/settings.json");

```

Comment:
This won't be the default location where vscode stores workspace settings: If you have the workspace open in `root` and the project in `root/project`, then this path I think would be `rooot/project/.vscode/settings.json`, while opening up the local workspace settings would open up `root/.vscode/settings.json`. Unless I'm misinterpreting where you intend to place the entry point that we run?